### PR TITLE
9 allow cors

### LINKED ISSUE #3: Put heroku links in README
**WHY**

This app is in prod, need to keep track of staging and prod links somewhere

**WHERE**

README

### LINKED ISSUE #9: CORS and localport issues
**WHY**

API request does not pass access control, a conflict related to CORS

**WHERE**

puma.rb
config/initializers/cors.rb

**HOW**

- [x] uncomment the CORS section
- [x] configurate the CORS section
- [x] set new localport for puma conflict, from 3000 to 8080
- [x] test locally

### PR DESCRIPTION
**Pull request by floriansr**
**Related issues**

closes #9 
closes #3 

**Changes**

➕ [add] heroku link into readme.md
:hammer_and_wrench: [fixup] uncomment CORS section, allow all origins
:hammer_and_wrench: [fixup] set new localport to 8080

* * *
- [x] I ran this code locally
- [x] I updated the README

### CODE REVIEW

File: config/puma.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
 # Specifies the `port` that Puma will listen on to receive requests; default is 3000.
 #
-port        ENV.fetch("PORT") { 3000 }
+port        ENV.fetch('PORT')
```
Comment (julienemo) [inline config/puma.rb:13]: This change is unnecessary

Comment (floriansr) [reply config/puma.rb:13]: why ?

Comment (julienemo) [reply config/puma.rb:13]: cors looks for ENV['PORT'] anyway, 3000 is the default value

File: config/initializers/cors.rb [hunk 2]
```diff
@@ -5,12 +5,13 @@
...
-#     origins 'example.com'
-#
-#     resource '*',
-#       headers: :any,
-#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
-#   end
-# end
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins '*'
```
Comment (julienemo) [inline config/initializers/cors.rb:10]: I suggest we don't allow * but only dev and prod links from our REACT app

**Pull request merged**